x-pack/plugins/security_solution/public/management/hooks/use_with_show_endpoint_responder.tsx

```diff
@@ -53,7 +52,15 @@ export const useWithShowEndpointResponder = (): ShowEndpointResponseActionsConso
                 endpointAgentId,
                 endpointCapabilities: endpointMetadata.Endpoint.capabilities ?? [],
                 endpointPrivileges,
-              }),
+              }).filter((command) =>
+                command.name !== 'status'
+                  ? isResponseActionSupported(
+                      'endpoint',
+                      getCommandKey(command.name as ConsoleResponseActionCommands),
+                      'manual'
+                    )
+                  : true
```

Comment:
Probably don't need this for endpoint. I just added this to test out the differences in commands available to `endpoint` vs `sentinel_one`

Reply:
True. does not hurt to leave it though so I'm ok if you leave it as is.